packages/utils/src/is.ts

```diff
@@ -41,7 +41,7 @@ function isBuiltin(wat: unknown, className: string): boolean {
  * @param wat A value to be checked.
  * @returns A boolean representing the result.
  */
-export function isErrorEvent(wat: unknown): boolean {
+export function isErrorEvent(wat: unknown): wat is ErrorEvent {
```

Comment:
Is `ErrorEvent` defined in node types? I wonder if this breaks some users setups.

Reply:
you are right, good catch, I guess it is not. I tried in a node TS app and it seems unavailable 😬 will remove this again!